generate/avro_template.go

```diff
@@ -93,6 +130,33 @@ func Test{{ .name }}_WriteAvroCF(t *testing.T) {
 		t.Error("OCF reports deleted true expected false")
 	}
 }
+
+func Example{{ .name }}BulkAvroWrite() {
+	input := []*{{ .name }}{ {}, {}, {} }
+	inputChan := make(chan *{{ .name }})
+
+	devnull, _ := os.Open("/dev/null")
+	defer devnull.Close()
+
+	errChan := {{ .name }}BulkAvroWriter(devnull, time.Now(), inputChan)
+
+	for _, item := range input {
+		select {
+		case err := <-errChan:
+			fmt.Print(err)
```

Comment:
wouldn't it be better to log?

Reply:
Yeah but it is an example, so I was just trying to express the minimum do something